### ISSUE TITLE
undefined reference to symbol 'pwgMediaForPWG'

### ISSUE THREAD

**Issue by abdelq** [michaelrsweet/lprint#80]
With [the commit](3a5aa5018bcaa14ef1440f731fc7fcca2a45f848) that edits configure to no longer reference libcups:
```
Linking lprint...
/usr/bin/ld: lprint-dymo.o: undefined reference to symbol 'pwgMediaForPWG'
/usr/bin/ld: /lib/arm-linux-gnueabihf/libcups.so.2: error adding symbols: DSO missing from command line
collect2: error: ld returned 1 exit status
make: *** [Makefile:151: lprint] Error 1
```

Unsurprisingly, it's happening in the CI too.

**Comment by zdohnal** [comment]
IIUC the direct dependency on libcups was removed due expectation that nothing in lprint depends on libcups, but ```pwgMediaForPWG()``` in dymo driver was overlooked.

Now we have two ways how to deal with it:

1. ignore the failure for now in lprint, define a wrapper function in PAPPL, release a new PAPPL release and once it reaches testing images, update lprint code
2. revert the commit with removing the dependency to fix the CI for now, and then do the fix in PAPPL, as the point 1. mentioned

IMHO the better is number 2.

**Comment by michaelrsweet** [comment]
The reason I removed it is to avoid getting the wrong libcups (PAPPL supports libcups2 and libcups3), which in my case at least was a very real problem.

**Comment by zdohnal** [comment]
@michaelrsweet I'm sorry, the PR #85 fixed #81, not #81 - I've messed up the number in the last commit message :(
